packages/clerk-js/src/utils/captcha/turnstile.ts

```diff
@@ -209,14 +225,18 @@ export const getTunstileToken = async (captchaOptions: {
       captchaError: e,
     };
   } finally {
-    if (widgetDiv) {
-      if (isInvisibleWidget) {
-        document.body.removeChild(widgetDiv as HTMLElement);
-      } else {
-        (widgetDiv as HTMLElement).style.display = 'none';
-      }
+    // cleanup
+    closeModal?.();
+    const invisibleWidget = document.querySelector(`.${CAPTCHA_INVISIBLE_CLASSNAME}`);
+    if (invisibleWidget) {
+      document.removeChild(invisibleWidget);
+    }
+    const visibleWidget = document.getElementById(CAPTCHA_ELEMENT_ID);
+    if (visibleWidget) {
+      visibleWidget.style.display = 'none';
     }
   }
+  console.log({ captchaToken, captchaWidgetTypeUsed: widgetTypeUsed.join(',') });
 
-  return { captchaToken, captchaWidgetTypeUsed: isInvisibleWidget ? 'invisible' : 'smart' };
+  return { captchaToken, captchaWidgetTypeUsed: widgetTypeUsed.join(',') };
```

Comment:
Echoing here again that I'm pretty sure that backend expects only one of the two values (`invisible/smart`) for the `captchaWidgetTypeUsed` param.

Reply:
I think the BE ignores this for now but I'll check again, in any case we need these for analytics so the BE will need to change 👍🏻

Reply:
We can introduce a new parameter if we want analytics on the BE.

Reply:
We say that the BE can get the strategy used from CF while verifying the token so I'm reverting this change